REQUIRE

```diff
@@ -3,3 +3,5 @@ LightGraphs 0.9.0
 JuMP 0.13.2
 MatrixDepot
 BlossomV 0.1
+Hungarian 0.1
```

Comment:
it would be nice to depend on Hungarian 0.2 which includes additional features and performance enhancement. 
```suggestion
Hungarian 0.2
```